expedited-procedure.md

```diff
@@ -14,56 +14,73 @@ Confirmed with a metric such as:
+Regular communication is essential. By sending regular updates the Web Team and HX stakeholders are able to manage and mitigate the impact of the issue. 
+
+* Post in the main [#expedite](https://holidayextras.slack.com/messages/expedite/) slack channel. This should trigger the bot and will ask you to confirm this is a new expedite.
+![](/images/expedited-procedure/new_expedite.png)
+
+* You will then be asked if you are the owner of the expedite. When confirming, this will create a new expedite channel with the current date to move all conversation away from the main channel.
+![](/images/expedited-procedure/owning_expedite.png)
 
-* Post in the main [#expedite](https://holidayextras.slack.com/messages/expedite/) slack channel. #expedite should be used for owner updates only, so encourage questions to be sent direct. Include when you'll next update in each post. 
-* If issue is complex enough consider a dedicated Slack channel (regular communication in the primary #expedite channel should still continue).
+* #expedite should be used for owner updates only, so encourage questions to be sent directly to the temporary dated channel. 
+* #expedite should have the customer impact clearly communicated and regularly updated to allow for wider business information.
+* Owner to include when you'll next update in each post in #expedite.
 * If the issue is going to effect deployments, post in the [#deployments](https://holidayextras.slack.com/messages/deployments/) Slack channel.
 
-Note that Germany and the Shift Manager may not be on Slack, so you should open a separate communcation channel with them where appropriate.
+Note that Germany and the Shift Manager may not be on Slack, so you should open a separate communcation channel with them where appropriate e.g. hangouts.
 
 ### Email
 
-There should be three key email communications sent during an expedite. 
+There should be two key email communications sent during an expedite. 
 
 * An incident report email to let affected teams/individuals know of the issue.
 * An email to confirm the end of the incident, with brief notes on the solution and ongoing issues.
-* An email after the post-mortem to include a full summary of the incident, solution and any learning outcomes.
+	* What happened technically? 
+	* What was the customer impact? 
+	* What was the resolution?
+	* What next?
 
-Email recipients should include our System Maintenance list.
+Email recipients should include those involved and the below:
+
+	shortbreaksweb@holidayextras.com
+	system.maintenance@holidayextras.com 
+	expedite@holidayextras.com 
+	shift.manager@holidayextras.com 
+	systems@holidayextras.de 
+	webit@holidayextras.com 
```

Comment:
This is a public repo so we wouldn't want to expose these emails. Can you put them in a Google doc and link to, per the out of hours contact details?